sdk/cognitiveservices/cognitiveservices-visualsearch/README.md

```diff
@@ -17,53 +17,64 @@ npm install @azure/cognitiveservices-visualsearch
 
 #### nodejs - Authentication, client creation and visualSearch images as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { VisualSearchClient, VisualSearchModels, VisualSearchMappers } from "@azure/cognitiveservices-visualsearch";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new VisualSearchClient(creds, subscriptionId);
-  const acceptLanguage = "testacceptLanguage";
-  const contentType = "testcontentType";
-  const userAgent = "testuserAgent";
-  const clientId = "testclientId";
-  const clientIp = "testclientIp";
-  const location = "westus";
-  const market = "testmarket";
-  const safeSearch = "Off";
-  const setLang = "testsetLang";
-  const knowledgeRequest = "testknowledgeRequest";
-  const image = new require("stream").Readable();
-  client.images.visualSearch(acceptLanguage, contentType, userAgent, clientId, clientIp, location, market, safeSearch, setLang, knowledgeRequest, image).then((result) => {
-    console.log("The result is:");
-    console.log(result);
+import {
+  VisualSearchClient,
+  VisualSearchModels
+} from "@azure/cognitiveservices-visualsearch";
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+
+async function main(): Promise<void> {
+  const visualSearchKey = process.env["visualSearchKey"] || "<visualSearchKey>";
+  const visualSearchEndPoint =
+    process.env["visualSearchEndPoint"] || "<visualSearchEndPoint>";
+  const cognitiveServiceCredentials = new CognitiveServicesCredentials(
+    visualSearchKey
+  );
+  const client = new VisualSearchClient(cognitiveServiceCredentials, {
+    endpoint: visualSearchEndPoint
   });
-}).catch((err) => {
-  console.error(err);
-});
-```
 
-#### browser - Authentication, client creation and visualSearch images as an example written in JavaScript.
+  const insightsToken =
+    "ccid_tmaGQ2eU*mid_D12339146CFEDF3D409CC7A66D2C98D0D71904D4*simid_608022145667564759*thid_OIP.tmaGQ2eUI1yq3yll!_jn9kwHaFZ";
```

Comment:
This shouldnt be hard coded

Reply:
It is a generic publicly available token taken from https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/cognitive-services/bing-visual-search/use-insights-token.md. 

I have removed it now